modules/issue_tracker/ajax/ToggleMyWatching.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Issue tracker
+ *
+ * Toggles the issue-watching flag for a user.
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Caitrin Armstrong <caitrin.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+/**
+ * Issue tracker
+ *
+ * Toggles the issue-watching flag for a user.
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Caitrin Armstrong <caitrin.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+if (!isset($_GET["issue_id"])) {
+    http_response_code(404);
+    echo json_encode(
+        array("error" => "Invalid issue id")
+    );
+    die();
+}
+    $issue_id = (int)$_GET["issue_id"];
+
+    $db   =& Database::singleton();
+    $user =& User::singleton();
+
+if (is_null($user) || is_null($user->getUsername())) {
+    http_response_code(401);
+    echo json_encode(
+        array("error" => "Please log in")
+    );
+}
+
+if ($db->pselectOne(
+    "
+        SELECT
+            EXISTS (
+                SELECT
+                    *
+                FROM
+                    issues_watching
+                WHERE
+                    issueID = :issue_id AND
+                    userID = :username
+            )
+    ",
+    array(
+     "username" => $user->getUsername(),
+     "issue_id" => $issue_id,
+    )
+)) {
+    $stmt = $db->prepare(
+        "
+            DELETE FROM
+                issues_watching
+            WHERE
+                issueID = :issue_id AND
+                userID = :username
+        "
+    );
+    $db->execute(
+        $stmt,
+        array(
+         "username" => $user->getUsername(),
+         "issue_id" => $issue_id,
+        )
+    );
+    echo json_encode(
+        array("watching" => false)
+    );
+} else {
+    if ($db->pselectOne(
+        "
+            SELECT
```

Comment:
phpcs has a problem with multiline strings. Not multiline sql queries.
It would still attempt to change it to:
```
some_function_call(
    "
        Some ugly indented
        multiline string.
    "
);
```

At least it leaves the formatting of the SQL query *inside* the string alone. All I ask is for readable SQL statements.